libtenzir/include/tenzir/importer.hpp

```diff
@@ -80,13 +80,11 @@ struct importer_state {
 
   measurement measurement_ = {};
   stopwatch::time_point last_report = {};
-  detail::heterogeneous_string_hashmap<uint64_t> schema_counters = {};
+  detail::flat_map<type, uint64_t> schema_counters = {};
```

Comment:
Since this map is cleared once every second there usually aren't many entries. Can't be the heterogeneous string map anymore because I need the actual type as the key instead of only its name.